cobalt/browser/h5vcc_system/h5vcc_system_impl.cc

```diff
@@ -52,10 +77,39 @@ void H5vccSystemImpl::GetLimitAdTracking(GetLimitAdTrackingCallback callback) {
   bool limit_ad_tracking = false;
 #if BUILDFLAG(IS_ANDROID)
   JNIEnv* env = base::android::AttachCurrentThread();
-  StarboardBridge* starbooard_bridge = StarboardBridge::GetInstance();
-  limit_ad_tracking = starbooard_bridge->GetLimitAdTracking(env);
+  StarboardBridge* starboard_bridge = StarboardBridge::GetInstance();
+  limit_ad_tracking = starboard_bridge->GetLimitAdTracking(env);
 #endif
   std::move(callback).Run(limit_ad_tracking);
 }
 
+void H5vccSystemImpl::GetUserOnExitStrategy(
+    GetUserOnExitStrategyCallback callback) {
+#if BUILDFLAG(IS_STARBOARD)
+  std::move(callback).Run(GetUserOnExitStrategyInternal());
+#elif BUILDFLAG(IS_ANDROID)
+  std::move(callback).Run(h5vcc_system::mojom::UserOnExitStrategy::kConceal);
+#else
+  NOTREACHED() << "Unsupported platform.";
+#endif
+}
+
+void H5vccSystemImpl::ConcealOrStop(ConcealOrStopCallback callback) {
+#if BUILDFLAG(IS_STARBOARD)
+  auto strategy = GetUserOnExitStrategyInternal();
+  if (strategy == h5vcc_system::mojom::UserOnExitStrategy::kConceal) {
+    SbSystemRequestConceal();
+  } else if (strategy == h5vcc_system::mojom::UserOnExitStrategy::kStop) {
+    SbSystemRequestStop(0);
+  }
+#elif BUILDFLAG(IS_ANDROID)
+  JNIEnv* env = base::android::AttachCurrentThread();
+  StarboardBridge* starboard_bridge = StarboardBridge::GetInstance();
+  starboard_bridge->RequestSuspend(env);
```

Comment:
nit: As we chatted today, could you leave a `TODO(): Investigate whether Android can support Exit in addition to Conceal, and double check this implementation`.